content/blog/2023-09-26-almalinux-jupyter.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "AlmaLinux for DataScience, Containers, and ARM"
+type: blog
+author: 
+ name: "Scott A. Williams - AKA vwbusguy"
+ bio: "MA, Senior Linux DevOps Engineer, University of California, Santa Barbara"
+ image: /users/vwbusguy.jpg
+date: '2023-09-26'
+post:
+  title: "Learn about how UCSB uses Almalinux in day-to-day operations."
+  image: /blog-images/23.09.26.ucsb_blog.svg
+---
+
+*Note: This post is a bit of a departure from what you normally see on the AlmaLinux blog, but we would love to share more stories like this! If you would like to write a post about how AlmaLinux helps you in your daily life, reach out to <a href="mailto:benny@almalinux.org">benny</a>.*
+
+The [College of Letters and Science at the University of California, Santa Barbara](https://lsit.ucsb.edu/) (UCSB) uses AlmaLinux, among other Linux distributions, in regular day-to-day operations. One such use case for AlmaLinux is the build infrastructure behind data science environments that serve hundreds to thousands of students per term.
+
+## Jupyter at UCSB
+
+Jupyter is used extensively at UCSB for classroom and research environments supporting Python and R language-driven research and curriculums, across such broad disciplines as data science, economics, linguistics, physics, psychology, and earth sciences. Traditionally, Jupyter and RStudio environments were either done in lab environments or on student machines. Unfortunately, this led to significant fragmentation of versioning and technical debt. Before long, machines were running old versions of things and were one well-meaning `pip install` or `install.packages()` away from fetching a library that was no longer compatible with older versions of the software. This means that student environments were easily broken, and TAs had to spend time fixing them and students sometimes had to learn on older, outdated versions of Python, R, and Jupyter.
```

Comment:
`Jupyter`

to

`[Jupyter](https://jupyter.org/)`

In the post new technologies are linked to. Adding hyperlink to keep it consistent.

Reply:
I missed adding a note for one other link: Terraform could be hyperlinked as well for the same reason.

Reply:
There's a later link for Jupyter, bit makes sense to link the first time it's mentioned. Adding one for TF